### PR TITLE
git ist nicht easy

### PR DESCRIPTION
**Pull request by ghost**
corrected print messages and language
fixed look, walk, continue query

### CODE REVIEW

File: src/Enemies.py [hunk 1]
```diff
@@ -35,3 +35,23 @@ def __iadd__(self):
...
+
+
+class Dragon(Enemy):
+    def __init__(self):
+        super().__init__(name="Giant Spider", hp=25, damage=2, armor=4)
+
+
+class Wolf(Enemy):
+    def __init__(self):
+        super().__init__(name="Spiders", hp=6, damage=3, armor=0)
```
Comment (Zeroks77) [inline src/Enemies.py]: This enemy should also be named wolve :P

File: src/Enemies.py [hunk 2]
```diff
@@ -35,3 +35,23 @@ def __iadd__(self):
...
+
+
+class Ghost(Enemy):
+    def __iadd__(self):
+        super().__init__(name="Trolls", hp=5, damage=10, armor=2)
```
Comment (Zeroks77) [inline src/Enemies.py]: this is a ghist and not a troll

File: src/Enemies.py [hunk 3]
```diff
@@ -35,3 +35,23 @@ def __iadd__(self):
...
+
+
+class Bat(Enemy):
+    def __init__(self):
+        super().__init__(name="Ogres", hp=1, damage=1, armor=1)
```
Comment (Zeroks77) [inline src/Enemies.py]: this is no orge

File: src/IdelOption.py [hunk 4]
```diff
@@ -16,17 +16,17 @@ def desciption(player):
         action = str(input(">Do you want to look, walk around, open you inventory or continue? \n"))
         action = re.sub("\n", "", action)
         action = re.sub("\s", "", action)
-        if action.upper() == "LOOK":
+        if action.upper() == "LOOK" or action.upper() == "look":
```
Comment (Zeroks77) [inline src/IdelOption.py:19]: for action.upper() the result will always be in CAPS
if ur input is look the result will be LOOK
if ur input is Look the result will be LOOK
therefor the or statement is unnessessary

Comment (Zeroks77) [reply src/IdelOption.py:19]: same for all others

**Review by Zeroks77 — CHANGES_REQUESTED** [review]
Resolve Comments

File: src/IdelOption.py (hunk 4 above)
Comment (ghost) [reply src/IdelOption.py:19]: alrighty

File: src/Enemies.py (hunk 3 above)
Comment (ghost) [reply src/Enemies.py]: 1

File: src/Enemies.py (hunk 2 above)
Comment (ghost) [reply src/Enemies.py]: 1

File: src/Enemies.py (hunk 1 above)
Comment (ghost) [reply src/Enemies.py]: 1 = will be changed after because keine lust jetzt

**Pull request merged**